app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cash = {}
+
+    def wrapper(*args, **kwargs) -> int:
+        for item in cash:
+            if f"{func}, {args}" in cash:
+                print("Getting from cache")
+                return cash[f"{func}, {args}"]
```

Comment:
There is no need to perform iteration, just check the presence of a particular key in dict and return the corresponding value accordingly.